tests/scripts/thread-cert/Cert_7_1_01_BorderRouterAsLeader.py

```diff
@@ -27,18 +27,28 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 #
 
+import functools
 import time
 import unittest
 
 import config
+import mle
```

Comment:
sort imports?

Reply:
Yeah. I divide the imports into several parts, some are related to the system, others are related to thread, the test cases, and I sorted each part separately.